commit 8c5c9e1a3a5a16d74fe2bc267b96afc71efcd4e4 (refs/changes/23/2523/7)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-11T00:13:39+00:00 (1 year, 11 months ago)
    
    HEAD-2316 Add sdc-volapi script to headnode tools
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index c1483da8..887fecbf 100644
--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,8 @@ SDC_ZONE_COMMANDS = \
 	sdc-vmapi \
 	sdc-waitforjob \
 	sdc-workflow \
-	updates-imgadm
+	updates-imgadm \
+	sdc-volapi
 
 SDC_ZONE_MAN_LINKS = \
 	$(SDC_ZONE_COMMANDS:%=$(PROTO)/opt/smartdc/man/man1/%.1)
